metricbeat/module/postgresql/activity/_meta/fields.yml

```diff
@@ -0,0 +1,80 @@
+- name: activity
+  type: group
+  description: >
+    One document per server process, showing information related to the current
+    activity of that process, such as state and current query.
+  fields:
+    - name: database_oid
+      type: long
+      description: >
+        OID of the database this backend is connected to.
+    - name: database_name
+      type: keyword
+      description: >
+        Name of the database this backend is connected to.
+    - name: pid
+      type: integer
+      description: >
+        Process ID of this backend.
+    - name: user.id
+      type: long
+      description: >
+        OID of the user logged into this backend.
+    - name: user.name
+      description: >
+        Name of the user logged into this backend.
+    - name: application_name
+      description: >
+        Name of the application that is connected to this backend.
+    - name: client.address
+      description: >
+        IP address of the client connected to this backend.
+    - name: client.hostname
+      description: >
+        Host name of the connected client, as reported by a reverse DNS lookup of client_addr.
+    - name: client.port
+      type: integer
+      description: >
+        TCP port number that the client is using for communication with this
+        backend, or -1 if a Unix socket is used.
+    - name: backend_start
```

Comment:
I don't like `start.*` since they are not different attributes of `start`. Maybe `backend.start`, `transaction.start`, etc. would work, but it feels strange to me.